tests/testsuite/metadata.rs

```diff
@@ -217,10 +217,19 @@ optional_feat = []
                 ],
                 "features": {
                   "default": [
+                    null,
+                    [
                       "default_feat"
+                    ]
                   ],
-                  "default_feat": [],
-                  "optional_feat": []
+                  "default_feat": [
```

Comment:
I am missing something, how is this a backwards compatible change to the metadata format?

Reply:
Bah. My fault - I remember explicitly implementing this compatibility. The commit is on the way.